### PR TITLE
Add service annotations to coderd service

### PR DESCRIPTION
**Pull request by deansheather**

### CODE REVIEW

File: templates/coderd.yaml [hunk 1]
```diff
@@ -240,6 +240,10 @@ metadata:
   name: {{ include "coder.serviceName" . }}
   {{- end }}
   namespace: {{ .Release.Namespace | quote }}
+  annotations:
+  {{- range $key, $value := .Values.coderd.serviceAnnotations }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
```
Comment (deansheather) [inline templates/coderd.yaml:246]: Is this OK if there's nothing in the map?

Comment (kylecarbs) [reply templates/coderd.yaml:246]: Should be

Comment (kylecarbs) [reply templates/coderd.yaml:246]: I'll try this

**Review by kylecarbs — APPROVED** [review]
This works, but you should remove the minimum Kube version added in #133. Seems like that isn't compatible with GKE.

**Pull request merged**